infrastructure/aws/cdk/config.py

```diff
@@ -42,7 +42,7 @@ class eoDBSettings(BaseSettings):
     instance_size: str = "SMALL"
     context: bool = True
     mosaic_index: bool = True
-
+    allocated_storage: int = 20
```

Comment:
@vincentsarago actually that's not going to work ; the idea is that we can merge this, run `cdk deploy` on the existing staging stack and no changes will happen. I see that the current DB storage is 100GiB (devseed account, us-east1-1, RDS instance id `eoapi-staging-eoapistagingpostgresdbf3d7e91a-cvyhomoxwlck`).

AWS is going to throw an error with 20 because you can't decrease an RDS storage allocation.

So I'll replace this with 100.

Reply:
🤷 default to 20 is fine. We can just overwrite when we need to update our stack